bundles/analysis/analyse/view/StartAnalyse.ol.js

```diff
@@ -1204,16 +1204,15 @@ Oskari.clazz.define('Oskari.analysis.bundle.analyse.view.StartAnalyse',
         _eligibleForAnalyse: function (layer) {
             return (((layer.hasFeatureData && layer.hasFeatureData()) ||
                 layer.isLayerOfType(this.contentPanel.getLayerType())) && 
-                    this._layerTypeIsEligibleForAnalyse(layer.getLayerType()) &&
-                    this._wfsLayerHasSupportedVersion(layer.getVersion()));
+                    this._layerHasSupportedVersion(layer.getLayerType(),layer.getVersion()));
         },
 
-        _layerTypeIsEligibleForAnalyse(layerType) {
-            return layerType === 'wfs' || layerType === 'analysislayer';
-        },
-
-        _wfsLayerHasSupportedVersion(layerVersion){
-            return layerVersion !== this._unsupportedWfsLayerVersion;
+        _layerHasSupportedVersion(layerType,layerVersion){
+            if(layerType === 'wfs' &&
+                layerVersion === this._unsupportedWfsLayerVersion){
```

Comment:
Should this include || layerType == 'analysis'?

Reply:
Idea in that function is that version is only checked with wfs layer type. With all other layer types (analysis and temp) version is not checked and function always returns true. If created analysis layer inherits version from source wfs layer i think there should not be version 3.0.0 analysis layers since wfs 3.0.0 layers cannot be added to analysis. I can add layerType == 'analysis'  as you stated to make it more clear and test.

Reply:
It also seems that i made false assumption that analysis layer inherits version from source wfs layer. When i created analysis layer from wfs 1.1.0 road accidents 2015 layer, version of created analysis layer was undefined in execution of that function. Nevertheless analysislayer is now added to the check.